File: solucoes/2021/nivel jr/fase 2/py/anagrama.py
N = int(input())
A = input().strip()
B = input().strip()

frase_a = ''.join(filter(str.isalpha, A)).lower()
frase_b = ''.join(filter(str.isalpha, B)).lower()

frase_ordA = ''.join(sorted(frase_a))
frase_ordB = ''.join(sorted(frase_b))

if frase_ordA == frase_ordB:
    print("S")
else: 
    print("N")

File: solucoes/2021/nivel jr/fase 2/js/anagrama.js
var N, A, B

scanf("%d%s%s", "N", "A", "B")

var frase_a = Array.from(A).filter(char => /[a-zA-Z]/.test(char)).join('')
var frase_b = Array.from(B).filter(char => /[a-zA-Z]/.test(char)).join('')

var frase_ordA = Array.from(frase_a).sort().join('');
var frase_ordB = Array.from(frase_b).sort().join('');

if (frase_ordA === frase_ordB) {
    console.log("S");
} else {
    console.log("N");
}
